src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -2897,8 +2897,9 @@ def squeeze():
 
         Slicing a single row from a single column will produce a single
         scalar DataFrame:
+        # TODO: SNOW-1372242: Remove instances of to_pandas when lazy index is implemented
 
-        >>> df_0a = df.loc[df.index < 1, ['a']]
+        >>> df_0a = df.loc[df.index.to_pandas() < 1, ['a']]
```

Comment:
Using the idea I described above would also give support to things like df.index < 1 (since it will just do it in native pandas)